### ISSUE TITLE
[Feature-Request] Same topics for gazebo simulated and real camera

### ISSUE THREAD

**Issue by flabrosse** [luxonis/depthai-ros#631]
Using the ready made package in ubuntu 22.04, I find that if I start a simulated camera (from gazebo classic) or a real camera, I get different topic names.  Gazebo produces:
```
/camera/color/camera_info
/camera/color/image_raw
/camera/color/image_raw/compressed
/camera/color/image_raw/compressedDepth
/camera/color/image_raw/theora
/camera/depth/camera_info
/camera/depth/image_raw/ffmpeg
/camera/depth/image_rect_raw
/camera/depth/image_rect_raw/compressed
/camera/depth/image_rect_raw/compressedDepth
/camera/depth/image_rect_raw/theora
/camera/depth/points
/camera/image_raw/ffmpeg
```
while the real camera gives us:
```
/oak/rgb/camera_info
/oak/rgb/image_raw
/oak/rgb/image_raw/compressed
/oak/rgb/image_raw/compressedDepth
/oak/rgb/image_raw/ffmpeg
/oak/rgb/image_raw/theora
/oak/stereo/camera_info
/oak/stereo/image_raw
/oak/stereo/image_raw/compressed
/oak/stereo/image_raw/compressedDepth
/oak/stereo/image_raw/ffmpeg
/oak/stereo/image_raw/theora
```

The urdf specification of the camera name as:
```
    <xacro:arg name="camera_name"   default="oak" />
```
seems to be ignored.

Could the gazebo plugin be made to match the topics of the real camera please?

**Comment by Serafadam** [comment]
Hi, we don't have (at least not yet) support for Gazebo and other simulators, although I think if you would use `rs_mode` you would get exact/pretty close topic names to those reported by Gazebo, more information on this mode can be found in docs

**Comment by flabrosse** [comment]
Hello,
Your answer did puzzle me, because I definitely get a simulated camera.  This made me look again at the gazebo files and to my embarrassment I realised that what creates a simulated 3D camera is indeed not your code but rather the use of a gazebo plugin that is loaded in another file.

My excuse is that I inherited the package I am working on and am still trying to fully explore it...  Not a very good excuse though.  I'll close this issue.
